.taskcluster.yml

```diff
@@ -75,9 +75,9 @@ tasks:
                   of ${chunk[2]}), run in the ${browser.channel} release of
                   ${browser.name}.
                 owner:
-                  $if: 'event.sender.login'
-                  then: ${event.sender.login}@users.noreply.github.com
-                  else: web-platform-tests@users.noreply.github.com
+                  $if: '"epochs" in event.ref'
```

Comment:
Can we make this `epochs/`; I don't think it matters, but it's closer to what we ideally want to check.

Reply:
Sure. I have updated it.